apiserver/pkg/storage/calico/resource.go

```diff
@@ -96,7 +96,7 @@ func CreateClientFromConfig() clientv3.Interface {
 	return c
 }
 
-// Versioned returns the versioned associated with this interface
+// Versioner returns the versioned associated with this interface
```

Comment:
Need to change the second "versioned" to "versioner".

Reply:
Thanks! Modified